add clusterclaim in concept (#313)
open-cluster-management-io/open-cluster-management-io.github.io: content/en/concepts/clusterclaim.md

---
title: ClusterClaim
weight: 2
---

<!-- spellchecker-disable -->

{{< toc >}}

<!-- spellchecker-enable -->

## What is ClusterClaim?

`ClusterClaim` is a cluster-scoped API created by users on a managed cluster, 

> and will be collected and saved into the status of the corresponding `ManagedCluster` on the hub.

Author: done

`ClusterCaim` is usually used to perform some extensible properties of the managed cluster like 

> clusterID, the version, vendor and cloud provider etc.

Author: done.

> clusterID, the version, vendor and cloud provider etc.

Author: thanks for the wording suggestions @dhaiducek

## Example 

> Here is a `ClusterClaim` example.

Reviewer: ```suggestion
Here is a `ClusterClaim` example specifying a clusterID:
```

Author: done

```
apiVersion: cluster.open-cluster-management.io/v1alpha1
kind: ClusterClaim
metadata:
  name: id.open-cluster-management.io
spec:
  value: 95f91f25-d7a2-4fc3-9237-2ef633d8451c
```

After applying the `ClusterClaim` above to any managed cluster, the value of the `ClusterClaim` 

> will be instantly reflected in the `ManagedCluster` on the hub cluster.

Reviewer: ```suggestion
is reflected in the `ManagedCluster` on the hub cluster:
```

Author: done

---
title: ClusterClaim
weight: 2
---

<!-- spellchecker-disable -->

{{< toc >}}

<!-- spellchecker-enable -->

## What is ClusterClaim?

> `ClusterClaim` is a cluster-scoped API available to users on a managed cluster. 

Reviewer 2: there are some reserved cluster name we need to mention

Author: done. introduced the reserved clusterclaims in the doc.

---
title: ClusterClaim
weight: 2
---

<!-- spellchecker-disable -->

{{< toc >}}

<!-- spellchecker-enable -->

## What is ClusterClaim?

`ClusterClaim` is a cluster-scoped API available to users on a managed cluster. 
The `ClusterClaim` objects are collected from the managed cluster and saved into 
the status of the corresponding `ManagedCluster` object on the hub.

## Usage

`ClusterCaim` is used to specify additional properties of the managed cluster like 
the clusterID, version, vendor and cloud provider. We defined some reserved `ClusterClaim` :

* `id.k8s.io`: A unique identifier for the managed cluster.
* `kubeversion.open-cluster-management.io`: The kubernetes version of the managed cluster.

> * `platform.open-cluster-management.io`: The platform where the managed cluster is running on, like AWS, GCE.

Reviewer 2: hrm maybe just id.k8s.io...

Author: ok. fixed.
